pkg/pxc/statefulset.go

```diff
@@ -132,3 +134,21 @@ func getConfigVolumes(cr *api.PerconaXtraDBCluster, cvName string) corev1.Volume
 	vol1.ConfigMap.Optional = &t
 	return vol1
 }
+
+// StatefulSetOwnerRef returns OwnerReference to statefulset
+func StatefulSetOwnerRef(sfs *appsv1.StatefulSet, scheme *runtime.Scheme) (metav1.OwnerReference, error) {
+	gvk, err := apiutil.GVKForObject(sfs, scheme)
+	if err != nil {
+		return metav1.OwnerReference{}, err
+	}
+
+	trueVar := true
+
+	return metav1.OwnerReference{
+		APIVersion: gvk.GroupVersion().String(),
+		Kind:       gvk.Kind,
+		Name:       sfs.GetName(),
+		UID:        sfs.GetUID(),
+		Controller: &trueVar,
+	}, nil
+}
```

Comment:
Why do we need this function? I don't see any references to it.

Reply:
Do you mean StatefulSetOwnerRef() from statefulset.go? 
Hm. It is not mine. Should I delete it?